made map as rank default (#132)
qiyunzhu/woltka: doc/cli.md

`--lineage` | Lineage strings. Can accept Greengenes-style rank prefix.
`--columns` | Table of classification units per rank (column).
`--map`, `-m` | Mapping of lower classification units to higher ones.

> `--map-as-rank/--map-no-rank` | Extract rank name from mapping filename. If omitted, will switch on when there are only mapping files.

Reviewer: ```suggestion
`--map-as-rank/--map-no-rank` | Extract rank name from mapping filename. On by default when classifying with only mapping files.
```
